common/src/main/scala/uk/ac/wellcome/models/MiroTransformable.scala

```diff
@@ -54,11 +54,20 @@ case class MiroTransformable(MiroID: String,
 
       // <image_title>: the Short Description.  This maps to our property
       // "label".
-      val label = miroData.title.get
+      //
+      // In the V collection, this is just a truncated form of the desc field,
+      // so we pass the full string as the label and omit a description.
```

Comment:
Worth mentioning that `.get` on an option her will lead to an exception if empty - this is expected here but might be useful to add a comment describing why.